.github/auto_assign.yml

```diff
@@ -0,0 +1,12 @@
+addReviewers: true
+addAssignees: author
+
+reviewers:
+  - dahlia
+  - longfin
+  - limebell
+  - earlbread
+  - riemannulus
+  - moreal
```

Comment:
Would the syntax for teams work?

```suggestion
- planetarium/libplanet
```

Reply:
Doesn't this disappear all of the reviewer signing when one person reviews it?

Reply:
Yes. I hear to @moreal.

I think insert `planetarium/libplanet` to newline for Visibility

<img width="731" alt="Screen Shot 2020-08-28 at 3 38 58 PM" src="https://user-images.githubusercontent.com/5278201/91529661-a3994400-e944-11ea-9af9-e954fdb8f685.png">